package.json

```diff
@@ -10,12 +10,14 @@
   },
   "dependencies": {
     "@babel/preset-typescript": "7.13.0",
+    "@popperjs/core": "2.9.2",
     "@rails/actioncable": "^6.0.0",
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.2",
-    "bootstrap": "4.5.2",
-    "bootstrap.native": "3.0.13",
+    "bootstrap": "5.0.2",
+    "bootstrap-icons": "1.5.0",
+    "bootstrap.native": "4.0.3",
```

Comment:
I think we don't need it. We used to use it to avoid `jQuery` but in `bootstrap` `5` they drop `jQuery` dependencies which is now full vanilla Javascript. What do you think?

Reply:
Good remark :) 
The only reason I kept BSN was that I wanted to reduce migration effort (as Bootstrap Native was already configured & is actually 100% build and maintained for Bootstrap 5).

Besides, BNS is not totally dead either: they now expose the advantage of being [faster, lighter and more compact](https://thednp.github.io/bootstrap.native/).

My opinion: for this project, I think we can keep as is. For another project, definitely we can use the JS provided by Bootstrap directly. 

Let me know if that's good to you @junan  :)

Reply:
Ok, fine for me